en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "Getting started with OVH PaaS Queue"
+categories: queue
+author: guillaumebreton
+lang: en
+---
+# OVH PaaS Queue and Kafka 101
+
+Welcome to the getting started of the OVH PaaS Queue. This guide will help you to understand the core concepts behind the OVH PaaS Queue
+and how to produce your first message to the platform and consume it.
+
+## What is the OVH PaaS Queue ?
+
+The OVH PaaS Queue provides you a mutualized queuing system based on [Apache Kafka](http://kafka.apache.org/).
+
+It powers fast, scalable and global low-latency messaging to allow you to connect anything to everything.
+
+It is a solution to send and receive messages between independent applications, to decouple systems and components or
+to connect, build, and scale realtime applications.
+
+## What is Kafka ?
+
+Kafka is a high-throughput, distributed, publish-subscribe messaging system.
+It allows you to connect producers and consumers through topics with fault tolerance as well as journalized events.
+All messages are persisted on disk and replicated within the cluster to prevent data loss.
+
+## What is a producer and a consumer ?
+
+Kafka maintains feeds of messages in topics. Producers write data to topics and
+consumers read from topics. Since Kafka is a distributed system, topics are partitioned
+and replicated across multiple nodes.
+Each message in a partition is assigned a unique offset. A consumer can save its offset position
+to remember the messages read.
+
+# Getting started
+
+The OVH PaaS Queue has been built so that you can connect painlessly to it.
+Topics are automatically created at the first request and a partitionning strategy is applied by default.
```

Comment:
"a partitioning strategy" --> typo + could you detail which strategy is deployed ? Tech details helps to get confidence in the tech